applications/dashboard/controllers/class.utilitycontroller.php

```diff
@@ -623,4 +623,16 @@ public function maintenance($updateMode = 0) {
         $this->deliveryMethod(DELIVERY_METHOD_JSON);
         $this->render();
     }
+
+    /**
+     * Redirect to touch icon.
+     *
+     * @since 1.0
+     * @access public
+     */
+    public function showTouchIcon() {
+        $redirect = Gdn_Upload::url(c('Garden.TouchIcon'));
+        redirect($redirect, 302);
+        exit();
```

Comment:
No. When you throw an exception, it stops executing that function. No code after it will be executed.